CPP_getResponseDemo.m

```diff
@@ -32,41 +33,35 @@
 % We set which keys are "valid", any keys other than those will be ignored
 expParameters.responseKey = {};
 
-
 %% init
 
 % Keyboard
 %  Make sure keyboard mapping is the same on all supported operating systems
 %  Apple MacOS/X, MS-Windows and GNU/Linux:
 KbName('UnifyKeyNames');
 
-
 % we ask PTB to tell us which keyboard devices are connected to the computer
 [cfg.keyboardNumbers, cfg.keyboardNames] = GetKeyboardIndices;
 
-cfg.keyboardNumbers
-cfg.keyboardNames
-
+cfg.keyboardNumbers;
```

Comment:
here I think we want to see these two printed in the command window, right? Then we could use:

```suggestion
fprintf('\n i% \n', cfg.keyboardNumbers);
```